samples/ServiceFabricRemoting/DatadogInstall.ps1

```diff
@@ -37,7 +37,7 @@ function Set-MachineEnvironmentVariable {
 
 Set-MachineEnvironmentVariable 'DD_DOTNET_TRACER_HOME' $DD_DOTNET_TRACER_HOME
 Set-MachineEnvironmentVariable 'DD_INTEGRATIONS' "$DD_DOTNET_TRACER_HOME\integrations.json"
-Set-MachineEnvironmentVariable 'DD_TRACE_LOG_PATH' "$LOGS_PATH\dotnet-tracer-native.log"
+Set-MachineEnvironmentVariable 'DD_TRACE_LOG_DIRECTORY ' "$LOGS_PATH"
```

Comment:
```suggestion
Set-MachineEnvironmentVariable 'DD_TRACE_LOG_DIRECTORY' "$LOGS_PATH"
```

Reply:
Not sure if the trailing whitespace would cause errors but...better to be safe than sorry 😄